delivery_price_method/tests/test_delivery_price_method.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 Trey, Kilobytes de Soluciones
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import Form
+from odoo.tests.common import SavepointCase
+from odoo.tools import float_compare
+
+
+class TestDeliveryPriceMethod(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        self = cls
+        product_shipping_cost = self.env["product.product"].create(
+            {
+                "type": "service",
+                "name": "Shipping costs",
+                "standard_price": 10,
+                "list_price": 100,
+            }
+        )
+        self.carrier = self.env["delivery.carrier"].create(
+            {
+                "name": "Test carrier",
+                "delivery_type": "fixed",
+                "product_id": product_shipping_cost.id,
+                "fixed_price": 99.99,
+            }
+        )
+        self.pricelist = self.env["product.pricelist"].create(
+            {
+                "name": "Test pricelist",
+                "item_ids": [],
```

Comment:
This was populated on purpose. Why are you removing it?

Reply:
This pricelist is put for avoiding weird behaviors when data pricelist has its currency changed (which happened in the past according to the installed modules). Thus, you should adjust the rest of the test code instead of removing this line.

Reply:
I think it works properly now, thank you.